starrocks/ddl.sql

```diff
@@ -1,4 +1,13 @@
 CREATE TABLE bluesky (
     `id` BIGINT AUTO_INCREMENT,
-    `data` JSON NOT NULL COMMENT "Primary JSON object, optimized for field access using FlatJSON"
-);
\ No newline at end of file
+    `data` JSON NOT NULL COMMENT "Primary JSON object, optimized for field access using FlatJSON",
+
+    sort_key VARBINARY AS encode_sort_key(
+        get_json_string(data, 'kind'),
+        get_json_string(data, 'commit.operation'),
+        get_json_string(data, 'commit.collection'),
+        get_json_string(data, 'did')
+    )
+) 
+DISTRIBUTED BY HASH(sort_key) BUCKETS 128
```

Comment:
The other databases in JSONBench (except Doris) also don't partition. Is partitioning a prerequisite for using Starrocks? If so, we can keep it, otherwise it would be nice to avoid it.